src/deepsparse/server/config.py

```diff
@@ -72,6 +111,10 @@ class EndpointConfig(BaseModel):
         default=1, description="The batch size to compile the model for."
     )
 
+    data_logging: Optional[
+        List[Dict[str, Union[List[MetricFunctionConfig], str]]]
```

Comment:
what is the purpose of the `str` in the union? is that handled in `build_logger`, let's remove if I'm not missing something